docs/graph/authentication/msal/MsalProvider.md

```diff
@@ -0,0 +1,68 @@
+---
+title: MsalProvider
+author: shweaver-MSFT
+description: Authentication provider based on the official Microsoft Authentication Library (MSAL).
+keywords: uwp, wpf, netstandard, windows, community, toolkit, graph, login, authentication, provider, providers, identity, msal
+dev_langs:
+  - csharp
+---
+
+# MsalProvider
+
+The MsalPRovider is an [IProvider](../IProvider.md) implementation built on the official Microsoft Authentication Library (MSAL). It is NetStandard 2.0 so it works in both UWP and WPF apps.
```

Comment:
```suggestion
The MsalProvider is an [IProvider](../IProvider.md) implementation built on the official Microsoft Authentication Library (MSAL). It is NetStandard 2.0 so it works in both UWP and WPF apps.
```

Typo fix here. Also minor nit. Do we really want a whole separate path/directory here for these two providers (msal/uwp)? It adds to the URL as well:

![image](https://user-images.githubusercontent.com/24302614/119880707-5ea93a80-bee1-11eb-92b4-1c94d01576b4.png)

So feels a bit redundant?

For the TOC would we want to just have Providers?>

- Providers
  - MsalProvider
  - WindowsProvider (UWP)

Also Windows Provider title is dupped here think you meant to say extensions or something?

![image](https://user-images.githubusercontent.com/24302614/119880926-9a440480-bee1-11eb-942c-936bbf27c1e7.png)